consensus/spos/consensusMessageValidator.go

```diff
@@ -457,6 +457,15 @@ func (cmv *consensusMessageValidator) checkMessageWithFinalInfoValidity(cnsMsg *
 	return nil
 }
 
+func (cmv *consensusMessageValidator) shouldNotVerifyLeaderSignature() bool {
+	if check.IfNil(cmv.consensusState.Header) {
```

Comment:
I think this changes the behavior for before enable epoch:
e.g header is not in consensusState, but we ignore leader signature missing on consensus message.

Reply:
added TODO comment